src/main/java/io/cdap/plugin/gcp/bigquery/sink/AbstractBigQuerySink.java

```diff
@@ -199,6 +199,8 @@ private Configuration getBaseConfiguration() throws IOException {
       baseConfiguration.set(BigQueryConstants.CONFIG_CLUSTERING_ORDER, getConfig().getClusteringOrder());
     }
     baseConfiguration.setStrings(BigQueryConstants.CONFIG_OPERATION, getConfig().getOperation().name());
+    baseConfiguration.setStrings(BigQueryConfiguration.OUTPUT_TABLE_WRITE_DISPOSITION_KEY,
+        getConfig().getWriteDisposition().name());
```

Comment:
nit: use indentation consistent with the rest of the class (aligned with BigQueryConfiguration.xxx on the previous line)

Reply:
Fixed!